src/components/plots/linePlot.ts

```diff
@@ -4,6 +4,8 @@ module Plottable {
 export module Plots {
   export class Line<X> extends XYPlot<X, number> {
 
+    public static PATH_CLASS = "line";
```

Comment:
Isn't this name going to cause a problem when we need a similar field on `Area`?